### PR TITLE
cli: show command on flynn ps

### LINKED ISSUE #3332: `flynn ps` should show command for each job
This would be especially useful to figure out where a `run` process came from.

### PR DESCRIPTION
**Pull request by imjorge**
closes #3332

```
$ flynn ps
ID                                          TYPE   COMMAND                   STATE  CREATED        RELEASE
host0-1b79af28-ae73-4b1f-b971-9a2c568cdfd0  web    /runner/init start web    up     5 minutes ago  905b99b3-edf0-496f-b9be-73442821f178
host0-ca18e56f-5fee-4f54-b943-2005168b3884  web    /runner/init start web    up     5 minutes ago  905b99b3-edf0-496f-b9be-73442821f178
host0-67952632-50db-4dba-878d-046990bda445  web    /runner/init start web    up     5 minutes ago  905b99b3-edf0-496f-b9be-73442821f178
host0-ecf97b0c-e29b-4675-bc1e-3661f3771954  clock  /runner/init start clock  up     5 minutes ago  905b99b3-edf0-496f-b9be-73442821f178

$ flynn ps -a
ID                                          TYPE         COMMAND                                                                                         STATE  CREATED         RELEASE
host0-1537acc9-2bda-4faf-9470-0e9c75c12bf9  slugbuilder  /tmp/builder/build.sh http://blobstore.discoverd/ea3b342c-4411-45df-9ab5-4a7502ec74a6/slug.tgz  down   13 minutes ago  3e5f9524-06a0-4440-b965-c772d9afedf0
host0-87418feb-c863-4f6c-ab85-56f441b460c4  web          /runner/init start web                                                                          down   12 minutes ago  8dde0ec1-8920-4936-a994-6c416aea96f5
host0-3bcb7d00-cfcc-4157-8875-983c2398270d  web          /runner/init start web                                                                          down   11 minutes ago  8dde0ec1-8920-4936-a994-6c416aea96f5
host0-9d8e3ec4-7cae-4799-9846-fc679ab3c514  web          /runner/init start web                                                                          down   11 minutes ago  8dde0ec1-8920-4936-a994-6c416aea96f5
host0-02eca4cb-430e-45d2-96b6-5275158a48bd  slugbuilder  /tmp/builder/build.sh http://blobstore.discoverd/46b48ee7-7f55-474d-b8b8-11d9ca54ba9b/slug.tgz  down   8 minutes ago   8dde0ec1-8920-4936-a994-6c416aea96f5
host0-95f6498c-8284-4cb3-833b-ca01243ab399  web          /runner/init start web                                                                          down   7 minutes ago   24cfdf54-ab0b-4f14-925f-bb095aeea096
host0-7f7c1698-5d2f-4093-a46a-2225b20e063f  web          /runner/init start web                                                                          down   7 minutes ago   24cfdf54-ab0b-4f14-925f-bb095aeea096
host0-82e416dd-dd2b-4777-9984-7111d069d9a7  web          /runner/init start web                                                                          down   7 minutes ago   24cfdf54-ab0b-4f14-925f-bb095aeea096
host0-6fc4450b-b390-41c3-818b-b10a0443a0f2  slugbuilder  /tmp/builder/build.sh http://blobstore.discoverd/40c4e7c8-3c55-4b71-a37f-43b7120d67ca/slug.tgz  down   5 minutes ago   24cfdf54-ab0b-4f14-925f-bb095aeea096
host0-1b79af28-ae73-4b1f-b971-9a2c568cdfd0  web          /runner/init start web                                                                          up     5 minutes ago   905b99b3-edf0-496f-b9be-73442821f178
host0-ca18e56f-5fee-4f54-b943-2005168b3884  web          /runner/init start web                                                                          up     5 minutes ago   905b99b3-edf0-496f-b9be-73442821f178
host0-67952632-50db-4dba-878d-046990bda445  web          /runner/init start web                                                                          up     5 minutes ago   905b99b3-edf0-496f-b9be-73442821f178
host0-ecf97b0c-e29b-4675-bc1e-3661f3771954  clock        /runner/init start clock                                                                        up     5 minutes ago   905b99b3-edf0-496f-b9be-73442821f178
```

### CODE REVIEW

**Comment by imjorge** [comment]
@lmars are CI errors related to this change?

**Comment by imjorge** [comment]
I run `go test ./...` locally and it reported no errors. So not sure how this is happening.

**Comment by josephglanville** [comment]
@imjorge that failure looks unrelated. That particular test is an integration test, some integration tests can be run locally with `make test-integration` but some (like this one) require a special cluster setup using the CI runner.

**Comment by lmars** [comment]
@imjorge we have pushed a bunch of fixes for intermittent CI failures recently, try rebasing on master to pull those into your branch.

File: docs/content/basics.md [hunk 1]
```diff
@@ -1,334 +1,369 @@
...
-## Run
-
-An interactive one-off process may be spawned in a new container:
-
-```
-$ flynn run bash
-```
-
-*See [here](/docs/cli#run) for more information on the `flynn run` command.*
+---
```
Comment (lmars) [inline docs/content/basics.md]: @imjorge any idea why this diff is the whole file? Did you edit on Windows (maybe you switched all the `LF` for `CRLF`)?

File: cli/ps.go [hunk 2]
```diff
@@ -21,23 +22,18 @@ Options:
...
-	host-d84dc657-83b8-4a62-aab8-ad97bb994761  web   down     2 minutes ago       cd698657-2955-4fa4-bc2f-8714b218a7a2
-	host-feaff633-a37b-4565-9ade-24bae0cfae03  web   down     About a minute ago  cd698657-2955-4fa4-bc2f-8714b218a7a2
-	host-e8b4f9be-e422-481f-928c-4c82f0bb5e8b  web   down     About a minute ago  cd698657-2955-4fa4-bc2f-8714b218a7a2
-	host-95747b4f-fdcd-4c44-ab72-b3d9609668e4  run   down     54 seconds ago      cd698657-2955-4fa4-bc2f-8714b218a7a2
-	host-cef95d8c-a632-4ae6-8a57-e13cbc24afa9  web   down     25 seconds ago      1b1db8ef-ba4d-4314-85c1-d5895a44b27e
-	host-ef6249e3-b463-4fea-a7dd-b1302872f821  web   down     20 seconds ago      1b1db8ef-ba4d-4314-85c1-d5895a44b27e
-	host-f25797dc-c956-4337-89af-d49eff50f58e  web   up       14 seconds ago      1b1db8ef-ba4d-4314-85c1-d5895a44b27e
-	6ec25d6e-2985-4807-8e64-02dc23c348bc       web   pending  7 seconds ago       1b1db8ef-ba4d-4314-85c1-d5895a44b27e
-	ab14754c-73b7-4212-a6d9-73b825587fd2       web   pending  2 seconds ago       1b1db8ef-ba4d-4314-85c1-d5895a44b27e
+$ flynn ps --all
```
Comment (lmars) [inline cli/ps.go]: You should keep the indentation to be consistent with all other examples

Comment (imjorge) [inline cli/ps.go]: done.

File: docs/content/basics.md [hunk 3]
```diff
@@ -48,43 +48,67 @@ It should also have added a default route of `example.demo.localflynn.com` point
 at the `example-web` service:
 
 ```
-ROUTE                             SERVICE      ID                                         STICKY  LEADER  PATH
-http:example.demo.localflynn.com  example-web  http/2e37467e-08fc-47e5-853b-4f0574cb6871  false   false   /
+ROUTE                          SERVICE      ID                                         STICKY  LEADER  PATH
+http:example.1.localflynn.com  example-web  http/5e47ae73-2812-4d67-8353-0a33007eb1d0  false   false   /
```
Comment (lmars) [inline docs/content/basics.md]: Keep this as `example.demo.localflynn.com` (that it what is referenced above).

File: docs/content/basics.md [hunk 4]
```diff
@@ -48,43 +48,67 @@ It should also have added a default route of `example.demo.localflynn.com` point
...
-DATABASE_URL=postgres://84abab8e4000453fe2e1ce3f4f04392a:80f9191af9ae6c7890e4caae54990255@leader.postgres.discoverd:5432/7f02ee75fe57cb70fe1e5d9afc37935c
+DATABASE_URL=postgres://6d534e4e0f63fafe37478302738075ec:74517b816771f56f58dc213e769a2c95@leader.postgres.discoverd:5432/33527719e7312be8b74b7974a1db0eeb
 FLYNN_POSTGRES=postgres
-PGDATABASE=7f02ee75fe57cb70fe1e5d9afc37935c
+PGDATABASE=33527719e7312be8b74b7974a1db0eeb
 PGHOST=leader.postgres.discoverd
-PGPASSWORD=80f9191af9ae6c7890e4caae54990255
-PGUSER=84abab8e4000453fe2e1ce3f4f04392a
+PGPASSWORD=74517b816771f56f58dc213e769a2c95
+PGUSER=6d534e4e0f63fafe37478302738075ec
```
Comment (lmars) [inline docs/content/basics.md]: Do we need to change all of these values?

File: cli/ps.go [hunk 5]
```diff
@@ -51,7 +47,7 @@ func runPs(args *docopt.Args, client controller.Client) error {
 	w := tabWriter()
 	defer w.Flush()
 
-	listRec(w, "ID", "TYPE", "STATE", "CREATED", "RELEASE")
+	listRec(w, "ID", "TYPE", "COMMAND", "STATE", "CREATED", "RELEASE")
```
Comment (lmars) [inline cli/ps.go]: As the command can be quite long, lets put it at the far right.

File: docs/content/basics.md (hunk 3 above)
Comment (imjorge) [inline docs/content/basics.md]: but that way it wont be consistent with output of `flynn route`.

File: docs/content/basics.md (hunk 4 above)
Comment (imjorge) [inline docs/content/basics.md]: changed to be consistent with the complete run.

File: cli/ps.go (hunk 5 above)
Comment (imjorge) [inline cli/ps.go]: k, will do.

**Comment by imjorge** [comment]
@lmars move `COMMAND` to the far right. updated outputs accordingly. let me know if you need something else.

File: docs/content/basics.md (hunk 3 above)
Comment (lmars) [inline docs/content/basics.md]: Drop this change, the rest of the example references `demo.localflynn.com` and there is no need to change it.

File: docs/content/basics.md [hunk 6]
```diff
@@ -101,8 +125,8 @@ Now the application is deployed, you can make HTTP requests to it using the
 default route for the application:
 
 ```
-$ curl http://example.demo.localflynn.com
-Hello from Flynn on port 8080 from container db0440f7-19b4-4369-b79e-7a48dba415c2
+$ curl http://example.1.localflynn.com
+Hello from Flynn on port 8080 from container 87418feb-c863-4f6c-ab85-56f441b460c4
```
Comment (lmars) [inline docs/content/basics.md]: Drop all these docs changes except for the addition of the `COMMAND` column in the output of `flynn ps` to keep the output consistent with deploying to a demo cluster.

**Comment by imjorge** [comment]
@lmars messed up the git log - pushed my changes on your last commit. need to figure out how to fix this.

**Comment by imjorge** [comment]
@lmars fixed

**Pull request merged**